src/main/java/fr/pilato/elasticsearch/crawler/fs/util/FsCrawlerUtil.java

```diff
@@ -221,6 +222,17 @@ public static LocalDateTime getCreationTime(File file) {
         return time;
     }
 
+    public static String getFileExtension(File file) {
+	String extension;
+        try  {
+ 	    extension = FilenameUtils.getExtension(file.getAbsolutePath());
+        } catch (Exception e) {
```

Comment:
Might be better to throw the Exception instead of swallowing it. I'd write something like:

```java
    public static String getFileExtension(File file) throws Exception {
	return FilenameUtils.getExtension(file.getAbsolutePath()).toLowerCase();
    }
```

This new method might deserve also some unit test in `FsCrawlerUtilTest`.